### ISSUE TITLE
cpio binary from busybox v1.22 segfaulting when used for extraction

### ISSUE THREAD

**Issue by vasishath** [omnirom/android_bootable_recovery#119]
When using cpio from busybox v1.22 (one in omni's repos) to extract a file, it segfaults. I have tried switching between clang and gcc but nothing helped. I am using latest recovery v3.0.3. I have a xiaomi MI3 (msm8974) device.

Can anyone confirm ?
Thanks

**Comment by russel5** [comment]
Yes, i had this too. But i had this only on printf function in modulecrcpatch, ie not see stdout in my logs, all other things seems to work.
Twrp 3.0.3, compilled on omni-7.1, phone - Z3C

**Comment by vasishath** [comment]
Strange thing is that if i use an older busybox binary of same version but compiled using 6.0 tree, that works fine.

**Comment by nailyk-fr** [comment]
I think it is related to the busybox built-in tool set.
